2020-11-09 21:43:13,965 ERROR: Exception on /index [GET] [in /home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/home/kriminel/nigel-blog/app/routes.py", line 36, in index
    return render_template('index.html', title='Home', posts=posts.items, form=form, next_url=next_url, prev_url=prev_url)
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask/templating.py", line 137, in render_template
    return _render(
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/kriminel/nigel-blog/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/kriminel/nigel-blog/app/templates/base.html", line 1, in top-level template code
    {% extends "bootstrap/base.html" %}
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 1, in top-level template code
    {% block doc -%}
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 4, in block "doc"
    {%- block html %}
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 20, in block "html"
    {% block body -%}
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 23, in block "body"
    {% block content -%}
  File "/home/kriminel/nigel-blog/app/templates/index.html", line 19, in block "content"
    {% include '_post.html' %}
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/kriminel/nigel-blog/app/templates/_post.html", line 12, in template
    said {{ moment.post.timestap).fromNow() }}:
jinja2.exceptions.TemplateSyntaxError: unexpected ')'
2020-11-09 21:43:15,228 ERROR: Exception on /index [GET] [in /home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/home/kriminel/nigel-blog/app/routes.py", line 36, in index
    return render_template('index.html', title='Home', posts=posts.items, form=form, next_url=next_url, prev_url=prev_url)
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask/templating.py", line 137, in render_template
    return _render(
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/kriminel/nigel-blog/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/kriminel/nigel-blog/app/templates/base.html", line 1, in top-level template code
    {% extends "bootstrap/base.html" %}
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 1, in top-level template code
    {% block doc -%}
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 4, in block "doc"
    {%- block html %}
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 20, in block "html"
    {% block body -%}
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 23, in block "body"
    {% block content -%}
  File "/home/kriminel/nigel-blog/app/templates/index.html", line 19, in block "content"
    {% include '_post.html' %}
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/kriminel/nigel-blog/venv/lib/python3.8/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/kriminel/nigel-blog/app/templates/_post.html", line 12, in template
    said {{ moment.post.timestap).fromNow() }}:
jinja2.exceptions.TemplateSyntaxError: unexpected ')'
